pixl_dcmd/tests/conftest.py

```diff
@@ -279,41 +218,15 @@ def mock_get(key, default) -> Optional[str]:
 
 
 @pytest.fixture()
-def vanilla_dicom_image_DX(row_for_dicom_testing) -> Dataset:
-    """
-    A DICOM image with diffusion data to test the anonymisation process.
-    Private tags were added to match the tag operations defined in the project config, so we can
-    test whether the anonymisation process works as expected when defining overrides.
-    The row_for_mri_dicom_testing dependency is to make sure the database is populated with the
-    project slug, which is used to anonymise the DICOM image.
-    """
-    return generate_dicom_dataset(Modality="DX")
-
-
-@pytest.fixture()
-def vanilla_single_dicom_image_DX(row_for_single_dicom_testing) -> Dataset:
+def vanilla_dicom_image_DX() -> Dataset:
     """
     A DICOM image with diffusion data to test the anonymisation process.
     Private tags were added to match the tag operations defined in the project config, so we can
     test whether the anonymisation process works as expected when defining overrides.
-    The row_for_single_dicom_testing dependency is to make sure the database is populated with the
-    project slug, which is used to anonymise the DICOM image.
     """
     return generate_dicom_dataset(Modality="DX")
 
 
-@pytest.fixture()
-def vanilla_dicom_image_MR(row_for_dicom_testing) -> Dataset:
```

Comment:
`generate_dicom_dataset` is now called directly in `test_anonymise_and_validate_dicom`, rather than creating fixtures for all different modalities we will support